packages/graph/behaviors/prefer-async.ts

```diff
@@ -0,0 +1,39 @@
+import { TimelinePipe } from "@pnp/core";
+import { InjectHeaders } from "@pnp/queryable";
+import { GraphQueryable, _GraphQueryable, graphGet } from "../graphqueryable.js";
+import { RichLongRunningOperation } from "@microsoft/microsoft-graph-types";
+
+export function PreferAsync(pollIntervalMs: number = 25000, maxPolls: number = 4): TimelinePipe {
+    return (instance: _GraphQueryable) => {
+        instance.using(InjectHeaders({ "Prefer": "respond-async" }));
+
+        instance.on.parse(async function (url, response, result) {
+            if (response.status === 202) {
```

Comment:
What happens if you get a different response? We have some standard logic [wrapped up here](https://github.com/pnp/pnpjs/blob/version-4/packages/queryable/behaviors/parsers.ts#L100) to check for errors. Used [here](https://github.com/pnp/pnpjs/blob/version-4/packages/queryable/behaviors/parsers.ts#L7) and [here](https://github.com/pnp/pnpjs/blob/version-4/packages/queryable/behaviors/parsers.ts#L23) for example.